src/encoded/static/libs/ui/modal.js

```diff
@@ -206,7 +206,7 @@ export const ModalFooter = ({
             submitBtn();
         }
         if (!dontClose) {
-            internalCloseModal();
+            internalCloseModal(e);
```

Comment:
e need to be set but not set in other places, so it can still crash. Does this ticket need to cover these too?

Reply:
Yeah, you’re absolutely right. I decided to reverse adding that stopPropagation call, as I didn’t actually need it anymore because I changed the way I was using modals from the cart page. I had kept the stopPropagation call, thinking was a general enhancement. But I need to think it through more if we need it in the future.